recipes/net-snmp/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package C)
+
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+conan_basic_setup(TARGETS KEEP_RPATHS)
```

Comment:
Ok for me.
Just a comment: due to `self.run(bin_path, run_environment=True)`, it doesn't really matter here. It's just that it would allow to omit `run_environment=True` on macOS.